src/api/app/views/webui2/webui/kiwi/images/_package_fields.html.haml

```diff
@@ -9,38 +9,40 @@
       = link_to_remove_association f do
         %i.fas.fa-times.text-danger
 
-%div{ 'tabindex': '-1', role: 'dialog', id: "#{f.object.name.present? ? 'package-' + f.object.name : 'add-package'}",
-class: "modal fade #{'new_element' if f.object.new_record?}" }
-  .modal-dialog{ role: 'document' }
-    .modal-content
-      .modal-header
-        %h5.modal-title #{f.object.name.present? ? 'Edit' : 'Add'} package
+  %div{ 'tabindex': '-1', role: 'dialog', id: "#{f.object.name.present? ? 'package-' + f.object.name : 'add-package'}",
+  class: "modal fade #{'new_element' if f.object.new_record?}" }
+    .modal-dialog{ role: 'document' }
+      .modal-content
+        .modal-header
+          %h5.modal-title #{f.object.name.present? ? 'Edit' : 'Add'} package
 
-      #flash-messages
-        %p.ui-state-error.alert-danger.p-3.mb-0.d-none
-          The name can not be empty!
+        #flash-messages
+          %p.ui-state-error.alert-danger.p-3.mb-0.d-none
+            The name can not be empty!
 
-      .modal-body
-        %p
-          = render partial: 'webui/autocomplete', locals: { html_id: :name, label: 'Name:'.html_safe,
```

Comment:
What is the reason for not using this partial anymore? I would prefer to keep using it if it's possible.

Reply:
Because in this case, the input field needs to be part of the cocoon construction, we can't pass a simple id.